<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1253</width>
    <height>813</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>File Manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/Asus/Downloads/File Manager.png</normaloff>C:/Users/Asus/Downloads/File Manager.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(111, 149, 182);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>1031</width>
      <height>761</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;
background-color: rgb(150, 150, 150);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="QA">
     <attribute name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/Icons/thunder.png</normaloff>:/Icons/thunder.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Quick Acess</string>
     </attribute>
     <widget class="QPushButton" name="obj_btn">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>111</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(&quot;3D.png&quot;);
text-align:justify;</string>
      </property>
      <property name="text">
       <string extracomment="3D Objects"/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Icons/3D.png</normaloff>Icons/3D.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="desktop_btn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>131</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string extracomment="Desktop"/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Icons/desktop.png</normaloff>Icons/desktop.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>160</width>
        <height>160</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="documents_btn">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>50</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string extracomment="Documents"/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Icons/documents.png</normaloff>Icons/documents.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>160</width>
        <height>160</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="downloads_btn">
      <property name="geometry">
       <rect>
        <x>512</x>
        <y>47</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string extracomment="Downloads"/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Icons/Downloads.png</normaloff>Icons/Downloads.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>160</width>
        <height>160</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="music_btn">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>50</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string extracomment="Music"/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Icons/music.png</normaloff>Icons/music.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>160</width>
        <height>160</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="picture_btn">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>240</y>
        <width>121</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string extracomment="Pictures"/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Icons/picture.png</normaloff>Icons/picture.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>160</width>
        <height>160</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="video_btn">
      <property name="geometry">
       <rect>
        <x>222</x>
        <y>240</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string extracomment="Videos"/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Icons/video.png</normaloff>Icons/video.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>160</width>
        <height>160</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="quick_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align:left;
border:0px;
color: rgb(255, 255, 255);
background-color: rgb(118, 118, 118);
border-radius:10px</string>
    </property>
    <property name="text">
     <string>Quick Acess</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/thunder.png</normaloff>Icons/thunder.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="home_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align:left;
border:0px;
color: rgb(255, 255, 255);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/home.png</normaloff>Icons/home.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="net_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align:left;
border:0px;
color: rgb(255, 255, 255);

border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Network</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/network.png</normaloff>Icons/network.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="recycle_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align:left;
border:0px;
color: rgb(255, 255, 255);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Recycle Bin</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/recycle-bin.png</normaloff>Icons/recycle-bin.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="settings_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align:left;
border:0px;
color: rgb(255, 255, 255);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/settings.png</normaloff>Icons/settings.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="theme_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>610</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align:left;
border:0px;
color: rgb(255, 255, 255);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Theme</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/theme.png</normaloff>Icons/theme.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="head_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align:left;
border:0px;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Quick Acess</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/thunder.png</normaloff>Icons/thunder.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="left_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
border:0px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/Icons/right.png</normaloff>:/Icons/right.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="right_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
border:0px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/Icons/left.png</normaloff>:/Icons/left.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>0</y>
      <width>411</width>
      <height>711</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;</string>
     </property>
     <property name="text">
      <string>Apply New Theme</string>
     </property>
    </widget>
    <widget class="QPushButton" name="th_1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Icons/darkorenge.png</normaloff>
       <normalon>Icons/darkorenge.png</normalon>Icons/darkorenge.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>110</width>
       <height>150</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="th_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>230</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Icons/darkblue.png</normaloff>Icons/darkblue.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>110</width>
       <height>150</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="th_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>350</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Icons/qdark.png</normaloff>Icons/qdark.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>110</width>
       <height>150</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="th_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>350</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Icons/ManjaroMix.png</normaloff>Icons/ManjaroMix.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>110</width>
       <height>150</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="hide_btn">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>660</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/Icons/up.png</normaloff>:/Icons/up.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>45</width>
       <height>45</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="th_5">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>470</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Icons/default.png</normaloff>Icons/default.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>110</width>
       <height>150</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="close_btn">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>0</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/close.png</normaloff>Icons/close.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="large_btn">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>0</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/Icons/large.png</normaloff>:/Icons/large.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="minimize_btn">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>0</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/Icons/minimize.png</normaloff>:/Icons/minimize.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
